config.go

```diff
@@ -268,6 +268,37 @@ func (c ColumnIgnoreConfig) IgnoredColumnsFor(schemaName, tableName string) map[
 	return columnsConfig
 }
 
+// CascadingPaginationColumnConfig to configure pagination columns to be
+// used. The term `Cascading` to denote that greater specificity takes
+// precedence.
+type CascadingPaginationColumnConfig struct {
+	// PerTable has greatest specificity and takes precedence over the other options
+	PerTable map[string]map[string]string // SchemaName => TableName => ColumnName
+
+	// FallbackColumn is a global default to fallback to and is more specific than the default fallback,
+	// which would be the PK
+	FallbackColumn string
+}
+
+// PaginationColumnFor is a helper function to retrieve the column name to paginate by
+func (c *CascadingPaginationColumnConfig) PaginationColumnFor(schemaName, tableName string) (string, bool) {
+	if c == nil {
+		return "", false
+	}
+
+	tableConfig, found := c.PerTable[schemaName]
+	if !found {
+		return "", false
+	}
+
+	column, found := tableConfig[tableName]
+	if !found {
+		return "", false
+	}
```

Comment:
Minor nit: probably can get rid of the boolean and just check if `column == ""`.